src/ring/middleware/authorization.clj

```diff
@@ -0,0 +1,84 @@
+(ns ring.middleware.authorization
+  (:refer-clojure :exclude [update-keys])
+  (:require
+    [clojure.string :as str]
+    [ring.util.parsing :as parsing]))
```

Comment:
Can you change this to use formatting from the Clojure Style Guide, i.e.

```
  (:require [clojure.string :as str]
            [ring.util.parsing :as parsing]))

Reply:
Done, but the style guide seems to suggest the line-wrapped variant I used as preferable: https://guide.clojure.style/#line-break-ns-declaration

Reply:
Ah, I see they've updated it. Thanks for telling me. I think we'll still go with the "good" approach, though, in order to be consistent with the rest of the Ring codebase.